pkg/build/api/v1/types.go

```diff
@@ -710,6 +780,10 @@ type BuildRequest struct {
 
 	// env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar `json:"env,omitempty"`
+
+	// TriggeredBy helps pass information about a trigger from the
```

Comment:
Make the description about how the end user sees this - "triggeredBy describes which triggers led to the most recent update to the build config, and contains information about those triggers"